### ISSUE TITLE
Permissions data ingest not picking up updated record

### ISSUE THREAD

**Issue by JosephMcArthur** [oaworks/discussion#1807]
**Describe the bug**

The following result: https://api.openaccessbutton.org/permissions/2053-2296 says the embargo is 12 months. However, all the data in the sheet says `0` months. It was updated long ago but is not updated in the API for some unknown reason. Note that in the [export file](https://docs.google.com/spreadsheets/d/19pDvOY5pge-C0yDSObnkMqqlMJgct3iIjPI2rMPLQEc/edit#gid=1668332243) `2053-2296` is listed as 0 months.

`updated: "31/03/2021",` is shown in the API while in the data `14/05/2021` can be seen as the latest update (I think Natalia re-updated it recently to try and get it to work).

**Expected behavior**

The data should've been updated so that the embargo is now 0 months.

**Additional context**

For Plan S.
cc @natalianorori.

**Comment by markmacgillivray** [comment]
On the new dev system this one already updated to 0 embargo months, some time in April. I just reran the import again and it picked up the latest change from 14th May as well.

I've noticed recently there's been a few queries about picking up changes to these sheets. Note, we only just finished the work to pick up from the 3 new sheets (as opposed to 1 overall sheet) last week. We've not yet confirmed if that can be released to live, so it's still not on live. The same old problems with sheets not updating will persist until we update live.
